test/test_distributions.py

```diff
@@ -2315,6 +2494,9 @@ def test_params_contains(self):
                         constraint = dist.params[name]
                     except KeyError:
                         continue  # ignore optional parameters
+                    except TypeError:
+                        continue
```

Comment:
Why would we accept a TypeError here?

Reply:
Sorry I believe this is to work around a bad suggestion of mine to define `TransformedDistribution.params = constraints.dependent`. I think it should actually be `TransformedDistribution.params = {}`. I've commented below.

Reply:
@lazypanda1 Can we remove the `except TypeError` as Adam suggests? The real fix is to set `TransformedDistribution.params = {}`.

Reply:
This still needs to be addressed